### ISSUE TITLE
About gan loss

### ISSUE THREAD

**Issue by zhangqianhui** [taesungp/contrastive-unpaired-translation#32]
Hello @taesungp  @junyanz 

https://github.com/taesungp/contrastive-unpaired-translation/blob/afdc8fb02742b4eef7724ca452fcee38016ee008/models/networks.py#L362

I see you use both vanilla gan  and nonsaturating with softplus

When you use  BCEWithLogitsLoss:

For real samples:

L_{r} = BCE(logits1, 1) + BCE(logits, 0)
        = -(1 * log(sigmoid(logits1)) + 0 * log(1 - sigmoid(logits1)) + 0 * log(sigmoid(logits2)) + 1 * (1 - sigmoid(logits2))) 
        = -(log(sigmoid(logits1)) + log(1 - sigmoid(logits2))))

For fake samples:

L_{f} = BCE(logits2, 1)
       = -(log(sigmoid(logits2))

Obviously, it is else nonsaturating loss.

Thus, I don't understand your  defination of for vanilla gan. Why do you both nonsaturating  gan with two different funtions?
Maybe I misunderstand.

**Comment by taesungp** [comment]
You are absolutely right about that. It was merely an artifact of combining old codes of CycleGAN with the new pieces of codes. Sorry for the confusion.
